tested/languages/c/config.json

```diff
@@ -45,10 +45,10 @@
     "uint32": "supported",
     "int64": "supported",
     "uint64": "supported",
-    "bigint": "reduced",
+    "bigint": "supported",
```

Comment:
How is big int supported in C? `bigint` is meant for arbitrary precision numbers (such as `BigInteger` in Java).

Reply:
I was mistaken about the exact meaning of bitint.